diff -Naur agg-2.5/font_freetype/Makefile.am agg-2.5.oden/font_freetype/Makefile.am
--- agg-2.5/font_freetype/Makefile.am	2006-10-09 06:06:54.000000000 +0200
+++ agg-2.5.oden/font_freetype/Makefile.am	2008-11-08 15:02:42.000000000 +0100
@@ -4,7 +4,7 @@
 agginclude_HEADERS = agg_font_freetype.h
 lib_LTLIBRARIES = libaggfontfreetype.la
 
-libaggfontfreetype_la_LDFLAGS = -version-info @AGG_LIB_VERSION@ @FREETYPE_LIBS@ 
+libaggfontfreetype_la_LDFLAGS = -version-info @AGG_LIB_VERSION@ @FREETYPE_LIBS@  ../src/libagg.la
 libaggfontfreetype_la_SOURCES = agg_font_freetype.cpp
 libaggfontfreetype_la_CXXFLAGS = -I$(top_srcdir)/include @FREETYPE_CFLAGS@ 
 endif
diff -Naur agg-2.5/src/platform/X11/Makefile.am agg-2.5.oden/src/platform/X11/Makefile.am
--- agg-2.5/src/platform/X11/Makefile.am	2006-12-11 01:59:45.000000000 +0100
+++ agg-2.5.oden/src/platform/X11/Makefile.am	2008-11-08 14:58:14.000000000 +0100
@@ -1,7 +1,7 @@
 if ENABLE_X11
 lib_LTLIBRARIES = libaggplatformX11.la
 
-libaggplatformX11_la_LDFLAGS = -version-info @AGG_LIB_VERSION@  -L@x_libraries@
+libaggplatformX11_la_LDFLAGS = -version-info @AGG_LIB_VERSION@  -L@x_libraries@ ../../libagg.la
 libaggplatformX11_la_SOURCES = agg_platform_support.cpp
 libaggplatformX11_la_CXXFLAGS =  -I$(top_srcdir)/include -I@x_includes@
 libaggplatformX11_la_LIBADD = -lX11
diff -Naur agg-2.5/src/platform/sdl/Makefile.am agg-2.5.oden/src/platform/sdl/Makefile.am
--- agg-2.5/src/platform/sdl/Makefile.am	2006-10-09 06:07:25.000000000 +0200
+++ agg-2.5.oden/src/platform/sdl/Makefile.am	2008-11-08 14:58:35.000000000 +0100
@@ -2,7 +2,7 @@
 
 lib_LTLIBRARIES = libaggplatformsdl.la
 
-libaggplatformsdl_la_LDFLAGS = -version-info @AGG_LIB_VERSION@
+libaggplatformsdl_la_LDFLAGS = -version-info @AGG_LIB_VERSION@ ../../libagg.la
 libaggplatformsdl_la_SOURCES = agg_platform_support.cpp
 libaggplatformsdl_la_CXXFLAGS = -I$(top_srcdir)/include @SDL_CFLAGS@
 libaggplatformsdl_la_LIBADD = @SDL_LIBS@
